corehq/apps/hqwebapp/static/cloudcare/less/formplayer-webapp/query.less

```diff
@@ -12,6 +16,7 @@
 }
 
 .flex-child-sidebar {
+  max-width: 400px;
```

Comment:
not: looks good to me, but did delivery mention them wanting this smaller? easy fix nonetheless